databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -2476,6 +2380,61 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn test_actuate_out_of_range() {
+        let broker = DataBroker::default();
+        let authorized_access = broker.authorized_access(&permissions::ALLOW_ALL);
+
+        authorized_access
+            .add_entry(
+                "Vehicle.Cabin.Infotainment.Navigation.Volume".to_owned(),
+                broker::DataType::Uint8,
+                broker::ChangeType::OnChange,
+                broker::EntryType::Actuator,
+                "Some funny description".to_owned(),
+                Some(broker::types::DataValue::Uint32(0)), // min
+                Some(broker::types::DataValue::Uint32(100)), // max
+                None,
+                None,
+            )
+            .await
+            .expect("Register datapoint should succeed");
+
+        let vss_id = authorized_access
+            .get_id_by_path("Vehicle.Cabin.Infotainment.Navigation.Volume")
+            .await
+            .expect(
+                "Resolving the id of Vehicle.Cabin.Infotainment.Navigation.Volume should succeed",
+            );
+        let vss_ids = vec![vss_id];
+
+        let (sender, _) = mpsc::channel(10);
+        let actuation_provider = Provider { sender };
+        authorized_access
+            .provide_actuation(vss_ids, Box::new(actuation_provider))
+            .await
+            .expect("Registering a new Actuation Provider should succeed");
+
+        let mut request = tonic::Request::new(ActuateRequest {
+            signal_id: Some(SignalId {
+                signal: Some(proto::signal_id::Signal::Path(
+                    "Vehicle.Cabin.Infotainment.Navigation.Volume".to_string(),
+                )),
+            }),
+            value: Some(Value {
+                typed_value: Some(proto::value::TypedValue::Uint32(200)),
+            }),
+        });
+
+        request
+            .extensions_mut()
+            .insert(permissions::ALLOW_ALL.clone());
+
+        let result_response = proto::val_server::Val::actuate(&broker, request).await;
+        assert!(result_response.is_err());
+        assert_eq!(result_response.unwrap_err().code(), tonic::Code::OutOfRange)
```

Comment:
![grafik](https://github.com/user-attachments/assets/20486bee-c389-4bae-9840-144693863271)
I think the last two commits here are incompatible / create merge conflicts. Either needs to be changed after the other is merged. tonic::Code::OutOfRange needs to be changed to InvalidValue

Reply:
Yes you are right, I branched from the branch `fix/doc_open_provider_stream` I need to change it to make it work